src/abstractdataframe/show.jl

```diff
@@ -437,6 +407,7 @@ NOTE: The value of `maxwidths[end]` must be the string width of
 - `eltypes::Bool = true`: Whether to print the column type
    under the column name in the heading. Defaults to `true`.
 - `rowid = nothing`: Used to handle showing `DataFrameRow`
+- `buffer`: buffer passed around to avoid reallocations in `ourstrwidth`
```

Comment:
```suggestion
- `rowid::Union{Integer, Nothing} = nothing`: Used to handle showing `DataFrameRow`
- `buffer::IOBuffer`: buffer passed around to avoid reallocations in `ourstrwidth`
```